scripts/travis/build-all-in-one-image.sh

```diff
@@ -13,6 +13,14 @@ export REPO=jaegertracing/all-in-one
 docker build -f cmd/all-in-one/Dockerfile -t $REPO:latest cmd/all-in-one
 export CID=$(docker run -d -p 16686:16686 -p 5778:5778 $REPO:latest)
 make integration-test
+
+if [ -z "$( docker ps | grep jaeger )" ] ; then
```

Comment:
I think this condition is too specific. If the test fails, `make` will return a non-zero exit code, and that's the condition to dump the logs.

Reply:
Do you mean `export code=$(make integration-test)`?
I ran it locally and the result turns out to be string instead of integer. 

I think this dump would help when golang panics during the test. If the test failed without panic, the log won't be very interesting to see.